### PR TITLE
feature/20 - Support event based increment behavior using kafka topics

### PR DESCRIPTION
**Pull request by davidmeirlevy**

### CODE REVIEW

File: apps/api/config.ts [hunk 1]
```diff
@@ -1,12 +1,62 @@
-import { existsSync } from 'node:fs';
+import { existsSync, readFileSync } from 'node:fs';
```
Comment (qballer) [inline apps/api/config.ts:1]: don't use sync, yes I know it's on startup. still.

Comment (davidmeirlevy) [reply apps/api/config.ts:1]: But it's on startup.. why not?

Comment (qballer) [reply apps/api/config.ts:1]: Because nodejs, async etc... sync api should really never be used.

**Review by qballer — APPROVED** [review]

**Comment by davidmeirlevy** [comment]
Note to self:
I didn't check the tenant ID of the account plan.

**Pull request merged**